src/marinade.ts

```diff
@@ -67,6 +67,29 @@ export class Marinade {
     return MarinadeReferralGlobalState.fetch(this)
   }
 
+  /**
+   * Fetch all the referral partners
+   */
+  async getReferralPartners(): Promise<MarinadeReferralPartnerState[]> {
```

Comment:
This works only with one of the 2 referral program versions. The accounts have different sizes. Maybe `getReferralPartners` and/or `getReferralPartnersV2`?

Reply:
Ah, you are right.